just install errors

so using the install shell script ala

```
curl --proto '=https' --tlsv1.3 -sSf https://just.systems/install.sh | bash -s -- --to /usr/local/bin
```

gets an error when the script runs specifically this block

```shell
if [ -z ${tag-} ]; then
  tag=$(curl --proto =https --tlsv1.3 -sSf https://api.github.com/repos/casey/just/releases/latest |
    grep tag_name |
    cut -d'"' -f4
  )
fi
```

will periodically but regularly result in a 403, which is causing some ci failures for us.

```shell
+ '[' -z ']'
[110](https://github.com/org/repo/actions/runs/3884593200/jobs/6627379737#step:4:115)
++ curl --proto =https --tlsv1.3 -sSf https://api.github.com/repos/casey/just/releases/latest
[111](https://github.com/org/repo/actions/runs/3884593200/jobs/6627379737#step:4:116)
++ grep tag_name
[112](https://github.com/org/repo/actions/runs/3884593200/jobs/6627379737#step:4:117)
++ cut '-d"' -f4
[113](https://github.com/org/repo/actions/runs/3884593200/jobs/6627379737#step:4:118)
curl: (22) The requested URL returned error: 403
[114](https://github.com/org/repo/actions/runs/3884593200/jobs/6627379737#step:4:119)
+ tag=
[115](https://github.com/org/repo/actions/runs/3884593200/jobs/6627379737#step:4:120)
Error: Process completed with exit code 1.
```

--- a/README.md
+++ b/README.md
@@ -274,6 +274,12 @@ export PATH="$PATH:$HOME/bin"
 just --help
 ```
 
+Note that `install.sh` may fail on GitHub actions, or in other environments
+where many machines share IP addresses. `install.sh` calls GitHub APIs in order
+to determine the latest version of `just` to install, and those API calls are
+rate-limited on a per-IP basis. To make `install.sh` more reliable in such
+circumstances, pass a specific tag to install with `--tag`.
+
 ### GitHub Actions
 
 With [extractions/setup-just](https://github.com/extractions/setup-just):

